runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -677,6 +680,40 @@ private void processInventoryLoot(String event, LootRecordType lootRecordType, I
 		}
 	}
 
+	private boolean processHerbSackLoot(int timestamp, String event, LootRecordType lootRecordType)
```

Comment:
Since this is specific to the herbiboar, this should be named as such.
```suggestion
	private boolean processHerbiboarHerbSackLoot(int timestamp, String event, LootRecordType lootRecordType)
```